processor/src/state.js

```diff
@@ -37,16 +37,16 @@ module.exports = class VehicleState {
    *
    * @param {string} vin The VIN number of the vehicle.
    * @param {Data[]} data
-   * @returns {string[]} The address(es) successfully set.
+   * @returns {Promise<string[]>} The address(es) successfully set.
    */
   async setVehicle(vin, data) {
     const address = addressFromVIN(vin);
     const vehicle = await this._loadVehicle(vin);
     return this.context.setState(
       {
-        [address]: this._serialize(data)
+        [address]: this._serialize(data),
```

Comment:
Unnecessary comma!!

Reply:
The second most necessary comma I've ever seen